specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/ManagedEnvironments.json

```diff
@@ -1448,6 +1448,10 @@
       "description": "Configuration of Open Telemetry traces",
       "type": "object",
       "properties": {
+        "includeDapr": {
```

Comment:
>includeDapr [](http://example.com/codeflow?start=9&length=11)

In place of boolean flags used for including traces you can convert this in an extendable enum for example traces with [Dapr, None] values. In this way in future version you could add other types of traces in a non-breaking way.

Reply:
Hi razvanbadea
Thank you for your feedback.
We had went through options your mentioned during the design phase and end up with current implementation.
Basically, we want deliver these system components signal options (dapr for trace, keda for metrics) to customer independently, so its by design